test/client/views/settings.js

```diff
@@ -10,51 +11,127 @@ var internals = {};
+
+                this.view.$el.find('.client').val('foo123').keyup();
+
+                this.view.$('.submit').click();
+            });
 
-        context('with a changed client id', function () {
+            it('sets the client id on the model', function () {
 
-            it('sets the client id as the channel on the model');
+                expect(this.view.settingsModel.get('clientId')).to.equal('foo123');
+            });
 
+            it('sets the client id as the channel on the model', function (){
+                expect(this.view.settingsModel.get('channel')).to.equal('foo123');
+            });
         });
 
-        it('hides itself');
+
+        it('hides itself', function(){
+
+            this.view.$el.find('.submit').click();
+
+            var viewHideSpy = sinon.spy(this.view, 'hide');
+
+            this.view.$el.find('.client').val('foo123').keyup();
+
+            this.view.$el.find('.submit').click();
+
+            expect(viewHideSpy).to.have.been.called;
+        });
 
     });
 
     describe('when cancel is clicked', function () {
 
-        it('resets the views client id back to original client id');
+        it('resets the views client id back to original client id', function () {
+
+            this.view.$el.find('.client').val('foo123').keyup();
+
+            this.view.$el.find('.cancel').click();
+
+            expect(this.view.model.get('clientId')).to.equal('bar123');
+        });
+
+        it('hides itself', function () {
+
+            var viewHideSpy = sinon.spy(this.view, 'hide');
+
+            this.view.$el.find('.cancel').click();
+
+            expect(viewHideSpy).to.have.been.called;
+        });
+
+        it('re-renders itself to reset it\'s display for the next time it\'s shown', function () {
 
-        it('hides itself');
+            var viewRenderSpy = sinon.spy(this.view, 'render');
 
-        it('re-renders itself to reset it\'s display for the next time it\'s shown');
+            this.view.$el.find('.cancel').click();
+
+            expect(viewRenderSpy).to.have.been.called;
+        });
 
     });
 
     describe('when the original settings model is changed', function () {
 
-        it('changes the view\'s model');
+        it('changes the view\'s model', function () {
+
+            this.view.settingsModel.set('clientId', '1');
+
+            this.view.settingsModel.set('channel', 'changedChannel');
+
+            expect(this.view.model.get('clientId')).to.equal('1');
+
+            expect(this.view.model.get('channel')).to.equal('changedChannel');
+        });
 
     });
 
     describe('#show', function () {
 
-        it('shows the view');
+        it('shows the view', function () {
+
+            this.view.show();
+
+            expect(this.view.$('.modal')).to.have.css('display', 'block');
+        });
 
     });
 
     describe('#hide', function () {
 
-        it('hides the view');
+        it('hides the view', function () {
+
+            this.view.show();
```

Comment:
Instead of calling `show` here, you can manually set the element's display to block. That way this unit test doesn't depend on a working version of the behavior `show`.